docs/environment.md

```diff
@@ -4,7 +4,86 @@ title: Environment
 description: The resulting environment of a pixi installation.
 ---
 
-# Environment variables
+# Environments
+Pixi is a tool to manage virtual environments.
+This document explains what an environment looks like and how to use it.
+
+## Structure
+A pixi environment is located in the `.pixi/envs` directory of the project.
+This location is **not** configurable as it is a specific design decision to keep the environments in the project directory.
+This keeps your machine and your project clean and isolated from each other, and makes it easy to clean up after a project is done.
+
+If you look at the `.pixi/envs` directory, you will see a directory for each environment, the `default` being the one that is used if you only have one environment.
```

Comment:
```suggestion
If you look at the `.pixi/envs` directory, you will see a directory for each environment, the `default` being the one that is used if you only have one environment, this will also be the case if you do not specify any environments.
```

Reply:
Rewrote the line alittle